CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.2.1] 2020-04-10
```

Comment:
IMO we should make a separate PR for the release just before we create that tag. Until then all changes should be made separately.